apps/web/lib/features/manual-time/add-manual-time-modal.tsx

```diff
@@ -47,40 +51,30 @@ export function AddManualTimeModal(props: IAddManualTimeModalProps) {
 		(e: FormEvent<HTMLFormElement>) => {
 			e.preventDefault();
 
-			const timeObject = {
-				date,
-				isBillable,
-				startTime,
-				endTime,
-				teamId,
+			const startedAt = new Date(date);
+			const stoppedAt = new Date(date);
```

Comment:
@CREDO23 I am not getting logic here... can you explain? Why date is same and only hours / minutes we set!? What is this for???

Reply:
The date will be the same @evereq :
![Screenshot from 2024-08-02 20-22-46](https://github.com/user-attachments/assets/552df608-ee4a-4931-8032-53b0445f68f1)

Reply:
Initially and When `Start Time` change, `End Time` must be must be 10 minutes longer than `Start Time`